### YamlMime:ManagedReference
items:
- uid: System.Windows.Data.ValueUnavailableException
  id: ValueUnavailableException
  children:
  - System.Windows.Data.ValueUnavailableException.#ctor
  - System.Windows.Data.ValueUnavailableException.#ctor(System.String)
  - System.Windows.Data.ValueUnavailableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Windows.Data.ValueUnavailableException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ValueUnavailableException
  nameWithType: ValueUnavailableException
  fullName: System.Windows.Data.ValueUnavailableException
  type: Class
  summary: "Исключение, вызванное &lt;xref:System.Windows.Data.BindingGroup.GetValue%28System.Object%2CSystem.String%29?displayProperty=fullName&gt; метод, если значение не доступно."
  syntax:
    content: 'public class ValueUnavailableException : SystemException'
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor
  id: '#ctor'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException()
  nameWithType: ValueUnavailableException.ValueUnavailableException()
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException()
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Initializes a new instance of the <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;></xref> class."
  remarks: "Этот конструктор инициализирует <xref:System.Exception.Message%2A>свойство нового экземпляра системным сообщением, которое описывает ошибку, такие как «системная ошибка».</xref:System.Exception.Message%2A> Это сообщение учитывает текущую культуру системы.       В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.Windows.Data.ValueUnavailableException>.</xref:System.Windows.Data.ValueUnavailableException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Пустая ссылка (`Nothing` в Visual Basic). |   | <xref:System.Exception.Message%2A>| Строка локализованного сообщения об. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ValueUnavailableException ();
    parameters: []
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException(String)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String)
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> класса с заданным сообщением."
  remarks: "Содержимое `message` должно быть понятно пользователю. Код, вызывающий этот конструктор, должен убедиться, что эта строка локализована для текущего языка и региональных параметров системы.       В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.Windows.Data.ValueUnavailableException>.</xref:System.Windows.Data.ValueUnavailableException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Пустая ссылка (`Nothing` в Visual Basic). |   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ValueUnavailableException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение, описывающее ошибку."
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException(SerializationInfo,StreamingContext)
  nameWithType: ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> сериализованными данными."
  syntax:
    content: protected ValueUnavailableException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Объект, содержащий сериализованные данные объекта."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Контекстные сведения об источнике или назначении."
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Windows.Data.ValueUnavailableException
  langs:
  - csharp
  name: ValueUnavailableException(String,Exception)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String,Exception)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String,Exception)
  type: Constructor
  assemblies:
  - PresentationFramework
  namespace: System.Windows.Data
  summary: "Инициализирует новый экземпляр <xref href=&quot;System.Windows.Data.ValueUnavailableException&quot;> </xref> класса с указанным сообщением об ошибке и ссылкой на внутреннее исключение, вызвавшее данное исключение."
  remarks: "Исключение, созданное как прямой результат предыдущего исключения, может содержать ссылку на предыдущее исключение в <xref:System.Exception.InnerException%2A>свойство.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Свойство возвращает то же значение, которое передается конструктору, или пустую ссылку (`Nothing` в Visual Basic) Если <xref:System.Exception.InnerException%2A>свойства не предоставляет конструктору значение внутреннего исключения.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       В следующей таблице показаны исходные значения свойств для экземпляра класса <xref:System.SystemException>.</xref:System.SystemException>      | Свойство | Значение |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Ссылка на внутреннее исключение. |   | <xref:System.Exception.Message%2A>| Строка сообщения об ошибке. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ValueUnavailableException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Сообщение об ошибке с объяснением причин исключения."
    - id: innerException
      type: System.Exception
      description: "Исключение, которое стало причиной текущего исключения. Если `innerException` параметр не является указателем null (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> в Visual Basic), текущее исключение сгенерировано в <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> блок, обрабатывающем внутреннее исключение."
  overload: System.Windows.Data.ValueUnavailableException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Windows.Data.ValueUnavailableException.#ctor
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException()
  nameWithType: ValueUnavailableException.ValueUnavailableException()
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException()
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String)
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException(String)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException(SerializationInfo,StreamingContext)
  nameWithType: ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Windows.Data.ValueUnavailableException.#ctor(System.String,System.Exception)
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException(String,Exception)
  nameWithType: ValueUnavailableException.ValueUnavailableException(String,Exception)
  fullName: System.Windows.Data.ValueUnavailableException.ValueUnavailableException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Windows.Data.ValueUnavailableException.#ctor*
  parent: System.Windows.Data.ValueUnavailableException
  isExternal: false
  name: ValueUnavailableException
  nameWithType: ValueUnavailableException.ValueUnavailableException
